documentation/en/source/pages/templates-spec/builders/builders-pxe.rst

```diff
@@ -0,0 +1,74 @@
+.. Copyright (c) 2007-2016 UShareSoft, All rights reserved
```

Comment:
A bug will be created on Jira in order to change update every copyrights. Would that be ok for you?